skbio/io/_registry.py

```diff
@@ -206,7 +188,9 @@ def wrapped_reader(fp, mode='U', mutate_fh=False, **kwargs):
                     else:
                         kwargs[file_arg] = None
 
-                with open_files(files, mode) as fhs:
+                # Do we want binary argument to come from the user
+                # (wrapped_reader) or from the reader (register_reader)?
```

Comment:
Almost certainly the reader, the user providing that information is actually actively unhelpful for the readers. This was a fundamental design flaw that I missed when I wrote the registry originally.